CHANGELOG.md

```diff
@@ -1,7 +1,71 @@
 # Changelog
 
 ## [0.15.0](https://github.com/brave/browser-laptop/releases/v0.15.0dev)
- - Update muon to 2.58.6. ([#8161](https://github.com/brave/browser-laptop/issues/8161))
+ - Added Widevine support for Amazon Prime Video. ([#5233](https://github.com/brave/browser-laptop/issues/5233))
+ - Added tear off tabs - Ability to drag tab into new window. ([#4402](https://github.com/brave/browser-laptop/issues/4402))
+ - Added user pinning in Brave Payments. ([#7347](https://github.com/brave/browser-laptop/issues/7347))
```

Comment:
"user pinning" doesn't make sense to me for this feature. Maybe "ledger site pining"